src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -138,4 +138,16 @@ public static boolean addBlobKey(String blobKey, Entity userEntity, DatastoreSer
  
     return true; 
   }
+
+  /**
+    adds a specified number of points to the user's points
+  */
+  public static void addPoints(Entity userEntity, int numPoints, DatastoreService datastore){
+    try {
+      userEntity.setProperty("score", ((int) userEntity.getProperty("score")) + numPoints);
```

Comment:
Instead of casting it as int i would cast it as a (Number) so that way you don't have to cast it as a long down below when you write your assert statements

Reply:
Okay, that's done